content/en/docs/collector/howto-custom-collector.md

```diff
@@ -0,0 +1,112 @@
+# HOW-TO: Build your own Collector instance
```

Comment:
Not a hugo expert and if this is mandatory, but most pages have a title like this:
```markdown
---
title: "Building your own collector instance"
weight: 30
---
```
Compare it with https://raw.githubusercontent.com/open-telemetry/opentelemetry.io/main/content/en/docs/collector/custom-auth.md -- I think there should also be some consistency in naming ("Building ..."

Reply:
Hi @svrnm, fixed on (7d3d01)[ https://github.com/open-telemetry/opentelemetry.io/pull/1277/commits/7d3d01bd356ff8c6ae72ffd52cc476f69108f7ca

Reply:
@rquedas , thanks, you can "resolve" this comment then